test/parallel/test-https-connect-address-family.js

```diff
@@ -36,8 +36,13 @@ function runTest() {
 }
 
 dns.lookup('localhost', {family: 6, all: true}, (err, addresses) => {
-  if (err)
+  if (err) {
+    if (err.code === 'ENOTFOUND') {
+      common.skip('localhost does not resolve to ::1');
```

Comment:
@thefourtheye Ah! That. OK, sorry, I misunderstood. Here's the answer, I think:

I tried something like that in 00f9b606388508b60ca09a25fcc029b2ac64d494

Sample CI run: https://ci.nodejs.org/job/node-stress-single-test/812/nodes=centos5-32/console:

```
not ok 1 parallel/test-https-connect-address-family
# 
# assert.js:89
#   throw new assert.AssertionError({
#   ^
# AssertionError: Could not find an IPv6 host for ::1
#     at Object.exports.fail (/home/iojs/build/workspace/node-stress-single-test/nodes/centos5-32/test/common.js:429:10)
#     at dns.lookup (/home/iojs/build/workspace/node-stress-single-test/nodes/centos5-32/test/parallel/test-https-connect-address-family.js:31:12)
#     at GetAddrInfoReqWrap.asyncCallback [as callback] (dns.js:65:16)
#     at GetAddrInfoReqWrap.onlookup [as oncomplete] (dns.js:79:17)
  ---
  duration_ms: 0.342
```

So even cycling through all those hostnames, we still need to account for hosts where none of them are configured.

So I think we can try it again after this lands, since that's basically what this does except that it only tries `localhost` and not all those other names.